src/request/components/request-detail-panel-logging.jsx

```diff
@@ -90,49 +92,27 @@ module.exports = React.createClass({
     render: function() {
         var request = this.props.request;
 
-        // get payloads 
-        var payload = getPayloads(request);
-        var logWriteMessages = payload.logWrite;
+        // get messages 
+        var payload = getMessages(request);
```

Comment:
I did this because it was the same as what we have elsewhere... I figure once they are the same we are going to be doing a refactor which will wash a lot of this out.